README.md

```diff
@@ -1,44 +1,60 @@
-
-# What components?
+# Eirini Components
 
 Eirini has the following components, the first two are available as subcommands of the `eirini` binary:
  
  - `Bifrost` converts and transfers cloud controller app specific requests to OPI specific objects and runs them in Kubernetes. It relies on the `Registry` to serve OCI images for droplets, and `OPI` to abstract the communication with Kube. 
- - `OPI` or the "orchestrator provider interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
+ - `OPI` or the "Orchestrator Provider Interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
  - `St8ger` implements Staging by running Kubernetes/OPI one-off tasks
+ - `Registry` provides a private docker registry that serves application OCI images created after staging the application droplet. While this used to be an Eirini component in the earlier versions of Eirini, the registry service is now offered as part of deploying [cloudfoundry-incubator/bits-service](https://github.com/cloudfoundry-incubator/bits-service) in the Eirini helm chart.
  
-# Tell me more 'bout OPI
-
-The really great thing about Diego is the high level abstractions above the level of containers and pods. Specifically, these are LRPs and Tasks. Actually, LRPs and Tasks are most of what you need to build both a PaaS and quite a lot of other things. And they're cross-cutting concepts that map nicely to all current orchestrators (for example to LRPs/Tasks directly in Diego, to Deployments/One-Off Tasks in Kube, and to Services and Containers in Swarm).
-
-One of the great things about Bosh is the CPI abstraction that lets it work on any IaaS. But so far Cloud Foundry has been tightly coupled to one specific Orchestrator (Diego). This was fine for fast iteration, but now orchestration is increasingly commodotised it makes a lot of sense to abstract ourselves away from the details of scheduling so an operator can use whatever orchestrator he or she wants and higher level systems can support all of them for free.
-
+# Orchestrator Platform Interface (OPI)
+
+The really great thing about Diego is the high level abstractions above the level of containers and pods.
+Specifically, these are Long Running Processes (LRPs) and Tasks.
+Actually, LRPs and Tasks are most of what you need to build a PaaS,
+and they're cross-cutting concepts that map nicely to all current orchestrators
+(for example to LRPs/Tasks directly in Diego,
+to Deployments/Jobs in Kube,
+and to Services/Containers in Swarm).
+
+One of the great things about BOSH is the CPI abstraction that lets it work on any IaaS.
+Cloud Foundry however, has been tightly coupled to one specific Orchestrator (Diego).
+This was fine for fast iteration, but now that orchestration is increasingly commoditized
```

Comment:
The "commoditized" statement is somewhat repeated from above, but we actually think that the whole sentence could be removed.